app/share/applications/org.fdroid.Repomaker.desktop

```diff
@@ -0,0 +1,12 @@
+[Desktop Entry]
+Type=Application
+Name=Repomaker
+GenericName=Repomaker
+Comment=Create your own F-Droid repositories
```

Comment:
Should be sent upstream.

Reply:
The whole file?

Reply:
Yeah, this file can be used by any distribution method, it can be translated too, it's not something specific to flatpak anyhow.

Reply:
Fixed in https://github.com/flathub/flathub/pull/771/commits/30651033054252e9f12437cbe4fe31a69c4dc67b.